webpack/ForemanInventoryUpload/Components/PageHeader/PageHeader.js

```diff
@@ -2,22 +2,26 @@ import React from 'react';
 import { Grid } from 'patternfly-react';
 import AutoUploadSwitcher from '../AutoUploadSwitcher';
 import InventoryFilter from '../InventoryFilter';
+import DocsButton from './components/DocsButton';
 import { INVENTORY_PAGE_TITLE } from '../../ForemanInventoryConstants';
 import './pageHeader.scss';
 
 const PageHeader = () => (
   <React.Fragment>
     <Grid.Row>
-      <Grid.Col xs={12}>
+      <Grid.Col xs={6}>
         <h1 className="inventory_title">{INVENTORY_PAGE_TITLE}</h1>
       </Grid.Col>
+      <Grid.Col xs={6}>
+        <AutoUploadSwitcher />
```

Comment:
moved upload switcher to the row above as it feels less dense with all of the other buttons